src/clientfactory.ts

```diff
@@ -42,11 +42,18 @@ export class DiscordClientFactory {
         this.botClient = new DiscordClient({
             fetchAllMembers: true,
```

Comment:
Having this set to `true` was causing issues with t2bot not starting up.

Reply:
The problem basically is that members is a priviliged intent and as soon as your bot is in > 100 guilds this setting will screw up the bot, if the bot owner does not go through all the hoops to get it whitelisted